### PR TITLE
Install missing COPYING

### PR DESCRIPTION
**Pull request by kenhys**
Put COPYING under /usr/share/groonga/COPYING

### CODE REVIEW

File: Makefile.am [hunk 1]
```diff
@@ -32,6 +31,7 @@ EXTRA_DIST =					\
 	gpg_uid					\
 	nginx_version				\
 	version-gen.sh
+dist_pkgdata_DATA = COPYING
```
Comment (kou) [inline Makefile.am]: Is `dist_` needed?

Comment (kenhys) [reply Makefile.am]: It seems that no need to `dist_` prefix.

Comment (kenhys) [reply Makefile.am]: Thanks.

**Comment by kenhys** [comment]
@kou Thanks,  I've removed needless prefix from it.

**Pull request merged**